my_weather_argparse.py

```diff
@@ -0,0 +1,22 @@
+from requests import get
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-c', '--cities', nargs='+')
+parser.add_argument('-u', '--units', help="enter f/c for Celsius or Fahrenheit. ")
+args = parser.parse_args()
+if args.units == 'f':
+    for city in args.cities:
+        weather = get(
+            "http://api.weatherstack.com/current?access_key=fce675c04b207f60a9e880fa544f7e80&query=" + city + "&units=f")
+        data = weather.json()
+        temperature = data["current"]["temperature"]
+        print("the weather in", city, "today is", temperature, "Fahrenheit")
+
+else:
+    for city in args.cities:
+        weather = get(
+            "http://api.weatherstack.com/current?access_key=fce675c04b207f60a9e880fa544f7e80&query=" + city)
+        data = weather.json()
+        temperature = data["current"]["temperature"]

```

Comment:
There is still no check for what happens if i give a city that doesn't exit.. the exercise requested:
3. If city name is not found:
http://api.weatherstack.com/current?access_key=KEY&query=lalalala
Print out an error that the city does not exists and offer other possible
cities (a few cities not really important which)

A "try except" block on this line can handle that case.
And that's another reason for this line to be outside of the if, so this try block is only written once.